README.md

```diff
@@ -18,6 +18,12 @@ machine and run the containers. You can disable installing Docker by setting
 ./runme.sh
 ```
 
+## Dev usage
+
+You can run `npm install` and `npm run` from the client and server folders as needed. They also each have a readme you should check out.
```

Comment:
Keep the existing formatting of this file. Lines are split at column 80 (max).

There's no server README, and it would be preferred to use the top-level `dev` NPM script, which runs `concurrently` for the frontend and the BFF. We use Node 16 for the project. The best way to get the most accurate results when testing is to build and run the container image.